src/modules/claims/claims.service.ts

```diff
@@ -149,14 +151,18 @@ export class ClaimsService {
         claim: {
             claimType: string;
             claimTypeVersion: number;
-            fields: { key: string; value: string | number }[];
+            fields?: { key: string; value: string | number }[];
```

Comment:
What is the difference between `requestorFields` and `fields` than?

Reply:
fields are left here temporarily, they are supposed to be removed after migration to requestor fields.